services/outputhost/kafkaStream.go

```diff
@@ -145,24 +151,23 @@ func GetDefaultKafkaMessageConverter(seqNo *int64, logger bark.Logger) KafkaMess
 		}
 
 		c.Message.Message = &store.AppendMessage{
-			SequenceNumber: common.Int64Ptr(atomic.AddInt64(seqNo, 1)),
+			Payload: &cherami.PutMessage{
+				Data: m.Value,
+				UserContext: map[string]string{
+					`key`:       string(m.Key),
+					`topic`:     m.Topic,
+					`partition`: strconv.Itoa(int(m.Partition)),
+					`offset`:    strconv.Itoa(int(m.Offset)),
+				},
+				// TODO: Checksum?
+			},
 		}
```

Comment:
(minor) why don't we set all of these members (c.Message, c.Message.Message, etc) inline .. as part of initialization in line 131 itself.